lib/evss/dependents/service.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module EVSS
+  module Dependents
+    class Service < EVSS::Service
+      configuration EVSS::Dependents::Configuration
+
+      def retrieve
+        perform(:get, 'load/retrieve')
```

Comment:
can you make another method or another class to cache this result since it's also used for prefill?

Reply:
In Redis, right?

Reply:
yea